src/content-scripts/player/components/EpisodeInfo.vue

```diff
@@ -1,7 +1,13 @@
 <template>
   <div class="EpisodeInfo as-space-y-4" :class="{ 'as-visible': isShowing }">
     <div class="as-flex as-flex-row as-items-center as-space-x-3">
-      <a v-if="showThemeLogo" class="as-pr-1" href="https://anime-skip.com">
+      <a
+        v-if="showThemeLogo"
+        class="as-pr-1"
+        href="https://anime-skip.com"
+        target="_blank"
+        :onclick="event => event.stopPropagation()"
```

Comment:
Is this needed?

Reply:
This stops the video from playing when you click the logo which makes sense to me as the new tab opens, you don't really want the anime you were watching to start playing... but if you want the video to play I can remove it no problem!

Reply:
Ahh, I see, it stops the pause. Also, with vue you can do `@click.stop` instead.

```suggestion
        @click.stop
```

Reply:
😮 that's awesome, thanks!!